docs/website/content/v0.6/en/guides/upgrading/index.md

```diff
@@ -26,10 +26,71 @@ In order to edit the control plane, we will need a working `kubectl` config.
 If you don't already have one, you can get one by running:
 
 ```bash
-talosctl kubeconfig
+talosctl --nodes <master node> kubeconfig
 ```
 
-### API Server
+### Automated Kubernetes Upgrade
+
+New command `talosctl upgrade-k8s` can be used to automate the Kubernetes upgrade process, this command was first released with Talos 0.6.1.
+
+```bash
+$ talosctl --nodes <master node> upgrade-k8s --from 1.18.6 --to 1.19.0
+updating pod-checkpointer grace period to "0m"
+sleeping 5m0s to let the pod-checkpointer self-scheckpoint be updated
```

Comment:
```suggestion
sleeping 5m0s to let the pod-checkpointer self-checkpoint be updated
```

Reply:
:facepalm: that's in the code, I'll fix it as well